### PR TITLE
Users can specify how many comments to load

### PR DESCRIPTION
**Pull request by AshleyLTW**
Added an input field on index.html for users to set the number of comments to display. This is by default 5.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -46,6 +46,8 @@ <h5>My hobbies</h5>
         <input type="text" name="comment" value="My comment is...">
         <input type="submit" value="Submit">
       </form>
+      Show me this many comments: <input type="number" id="numComments" value="5" onchange="filterComments(this.value)">
+
```
Comment (darrnshn) [inline portfolio/src/main/webapp/index.html:50]: nit: remove extra blank lines

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 // Copyright 2019 Google LLC
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
+// Licensed under the Apache License, Version 2.0 (the 'License');
```
Comment (darrnshn) [inline portfolio/src/main/webapp/script.js:3]: nit: best to leave the copyright as is

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -37,8 +37,15 @@ async function getComments() {
   }
 }
 
+function filterComments(value) {
+  var commentLimit = parseInt(value);
```
Comment (darrnshn) [inline portfolio/src/main/webapp/script.js:41]: nit: const

Comment (darrnshn) [inline portfolio/src/main/webapp/script.js:41]: I think "modern JS" suggests calling Number: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -37,8 +37,15 @@ async function getComments() {
...
+  if (commentLimit >= 0 ) {
```
Comment (darrnshn) [inline portfolio/src/main/webapp/script.js:42]: nit: no space between `0` and `)`

**Review by darrnshn — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (AshleyLTW) [reply portfolio/src/main/webapp/script.js:3]: Whoops should have checked my replace all a bit more carefully

**Pull request merged**